keyboards/dztech/dz40rgb/dz40rgb.c

```diff
@@ -0,0 +1,311 @@
+    {0, E_3,    D_3,    F_3},
+    {0, E_2,    D_2,    F_2},
+    {0, E_1,    D_1,    F_1},
+
+    {0, E_13,   D_13,   F_13},
+    {0, E_14,   D_14,   F_14},
+
+    {0, B_12,   A_12,   C_12},
+    {0, B_11,   A_11,   C_11},
+    {0, B_10,   A_10,   C_10},
+    {0, B_9,    A_9,    C_9},
+    {0, B_8,    A_8,    C_8},
+    {0, B_13,   A_13,   C_13},
+    {0, B_5,    A_5,    C_5},
+    {0, B_4,    A_4,    C_4},
+    {0, B_3,    A_3,    C_3},
+    {0, B_2,    A_2,    C_2},
+    {0, B_1,    A_1,    C_1}
+
+};
+
+const rgb_led g_rgb_leds[DRIVER_LED_TOTAL] = {
+
+    {{0|(11<<4)},   {20.36*11,  0}, 1},
+    {{0|(10<<4)},   {20.36*10,  0}, 1},
+    {{0|(9<<4)},    {20.36*9,  0}, 1},
+    {{0|(8<<4)},    {20.36*8,  0}, 1},
+    {{0|(7<<4)},    {20.36*7,  0}, 1},
+	{{0|(6<<4)},    { 20.36*6,  0}, 1},
+    {{0|(5<<4)},    { 20.36*5,  0}, 1},
+    {{0|(4<<4)},    { 20.36*4,  0}, 1},
+    {{0|(3<<4)},    { 20.36*3,  0}, 1},
+    {{0|(2<<4)},    { 20.36*2,  0}, 1},
+    {{0|(1<<4)},    { 20.36*1,  0}, 1},
+    {{0|(0<<4)},    { 20.36*0,  0}, 1},
+
+    {{0|(12<<4)},   {20.36*11, 21.33*0.5}, 1},
+    {{0|(13<<4)},  {20.36*0,21.33*0.5}, 1},
+
+    {{1|(11<<4)},   {20.36*11,  21.33}, 1},
+    {{1|(10<<4)},   {20.36*10,  21.33}, 0},
+    {{1|(9<<4)},    {20.36*9,  21.33}, 0},
+    {{1|(8<<4)},    {20.36*8,  21.33}, 0},
+    {{1|(7<<4)},    {20.36*7,  21.33}, 0},
+	{{1|(6<<4)},    { 20.36*6,  21.33}, 0},
+    {{1|(5<<4)},    { 20.36*5,  21.33}, 0},
+    {{1|(4<<4)},    { 20.36*4,  21.33}, 0},
+    {{1|(3<<4)},    { 20.36*3,  21.33}, 0},
+    {{1|(2<<4)},    { 20.36*2,  21.33}, 0},
+    {{1|(1<<4)},    { 20.36*1,  21.33}, 0},
+    {{1|(0<<4)},    { 20.36*0,  21.33}, 1},
+
+    {{1|(12<<4)},   {20.36*11, 21.33*1.5}, 1},
+    {{1|(13<<4)},  {20.36*0,21.33*1.5}, 1},
+
+    {{2|(11<<4)},   {20.36*11,  21.33*2}, 1},
+    {{2|(10<<4)},   {20.36*10,  21.33*2}, 0},
+    {{2|(9<<4)},    {20.36*9,  21.33*2}, 0},
+    {{2|(8<<4)},    {20.36*8,  21.33*2}, 0},
+    {{2|(7<<4)},    {20.36*7,  21.33*2}, 0},
+	{{2|(6<<4)},    { 20.36*6,  21.33*2}, 0},
+    {{2|(5<<4)},    { 20.36*5,  21.33*2}, 0},
+    {{2|(4<<4)},    { 20.36*4,  21.33*2}, 0},
+    {{2|(3<<4)},    { 20.36*3,  21.33*2}, 0},
+    {{2|(2<<4)},    { 20.36*2,  21.33*2}, 0},
+    {{2|(1<<4)},    { 20.36*1,  21.33*2}, 0},
+    {{2|(0<<4)},    { 20.36*0,  21.33*2}, 1},
+
+    {{2|(12<<4)},   {20.36*11, 21.33*2.5}, 1},
+    {{2|(13<<4)},  {20.36*0,21.33*2.5}, 1},
+
+    {{3|(11<<4)},   {20.36*11,  21.33*3}, 1},
+    {{3|(10<<4)},   {20.36*10,  21.33*3}, 1},
+    {{3|(9<<4)},    {20.36*9,  21.33*3}, 1},
+    {{3|(8<<4)},    {20.36*8,  21.33*3}, 1},
+    {{3|(7<<4)},    {20.36*7,  21.33*3}, 1},
+	{{3|(6<<4)},    { 20.36*5.5,  21.33*3}, 1},
+    {{3|(4<<4)},    { 20.36*4,  21.33*3}, 1},
+    {{3|(3<<4)},    { 20.36*3,  21.33*3}, 1},
+    {{3|(2<<4)},    { 20.36*2,  21.33*3}, 1},
+    {{3|(1<<4)},    { 20.36*1,  21.33*3}, 1},
+    {{3|(0<<4)},    { 20.36*0,  21.33*3}, 1}
+
+};
+#endif
+
+void bootmagic_lite(void)
```

Comment:
This is not actually needed.  It's in the core QMK code.

Reply:
I marked this as unresolved, since what you're doing here is identical to the stock code:

https://github.com/qmk/qmk_firmware/blob/c1c5922aae7b60b7c7d13d3769350eed9dda17ab/quantum/quantum.c#L1021-L1048

The only difference here, is that it is using variables in some cases, but should come back identical to what you have. 

If you want to leave this here, that's perfectly fine. but I'm not sure it's needed here.